src/components/Home.tsx

```diff
@@ -112,8 +121,9 @@ const Home: React.FC = () => {
                 addNotification("AddExpense sucess", "success");
 
                 setCurrentValue({
-                    ...initialState,
-                    accountIdx: currentValue.accountIdx,
+                    ...currentValue,
```

Comment:
I think the previous code is more reasonable. After we send the transaction, we reset to initial state accept the current wallet index